File: src/main/java/com/github/Doomsdayrs/Jikan4java/types/Support/Reviews/Anime/AnimeScore.java
package com.github.Doomsdayrs.Jikan4java.types.Support.Reviews.Anime;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.github.Doomsdayrs.Jikan4java.types.Support.Reviews.Scores;

/*
 * Licensed to the Apache Software Foundation (ASF) under one
 * or more contributor license agreements.  See the NOTICE file
 * distributed with this work for additional information
 * regarding copyright ownership.  The ASF licenses this file
 * to you under the Apache License, Version 2.0 (the
 * "License"); you may not use this file except in compliance
 * with the License.  You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 * ====================================================================
 * Jikan4java
 * 30 / December / 2018
 *
 * @author github.com/doomsdayrs
 */
public class AnimeScore extends Scores {
    @JsonProperty("overall")
    public int overall;

    @JsonProperty("story")
    public int story;

    @JsonProperty("animation")
    public int animation;

    @JsonProperty("sound")
    public int sound;

    @JsonProperty("character")
    public int character;

    @JsonProperty("enjoyment")
    public int enjoyment;
    
    @Override
    public String toString() {
        return "AnimeScore{" +
                "overall=" + overall +
                ", story=" + story +
                ", animation=" + animation +
                ", sound=" + sound +
                ", character=" + character +
                ", enjoyment=" + enjoyment +
                '}';
    }
}

File: src/commonMain/kotlin/com/github/doomsdayrs/jikan4java/data/model/support/reviews/anime/AnimeScore.kt
package com.github.doomsdayrs.jikan4java.data.model.support.reviews.anime

import com.github.doomsdayrs.jikan4java.data.model.support.reviews.Scores
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/*
 * This file is part of Jikan4java.
 *
 * Jikan4java is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Jikan4java is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Jikan4java.  If not, see <https://www.gnu.org/licenses/>.
 * ====================================================================
 * Jikan4java
 * 30 / December / 2018
 *
 * @author github.com/doomsdayrs
 */
@Serializable
data class AnimeScore(
	@SerialName("overall") override val overall: Int = 0,
	@SerialName("story") override val story: Int = 0,
	@SerialName("animation") val animation: Int = 0,
	@SerialName("sound") val sound: Int = 0,
	@SerialName("character") override val character: Int = 0,
	@SerialName("enjoyment") override val enjoyment: Int = 0
) : Scores